packages/desktop/views/dashboard/index.js

```diff
@@ -3,5 +3,6 @@ export * from './developer'
 export * from './settings'
 export * from './wallet'
 export * from './governance'
+export * from './send-flow'
```

Comment:
This isn't a collection of views like the others here are, this should be in a `components/popups/send` directory IMO